app/payment/session.go

```diff
@@ -76,6 +90,30 @@ func GetPayChs(s perun.SessionAPI) []PayChInfo {
 	return payChInfos
 }
 
+// SubPayChProposals registers a subscription for payment channel proposals.
+func SubPayChProposals(s perun.SessionAPI, notifier PayChProposalNotifier) error {
+	return s.SubChProposals(func(notif perun.ChProposalNotif) {
+		balsBigInt := notif.Proposal.InitBals.Balances[0]
+		notifier(PayChProposalNotif{
+			ProposalID:       notif.ProposalID,
+			Currency:         notif.Currency,
+			OpeningBals:      balsFromBigInt("ETH", balsBigInt, notif.Parts),
+			ChallengeDurSecs: notif.Proposal.ChallengeDuration,
+			Expiry:           notif.Expiry,
+		})
+	})
+}
+
+// UnsubPayChProposals unregisters a subscription for payment channel proposals.
```

Comment:
```suggestion
// UnsubPayChProposals unsubscribe a subscription for payment channel proposals.
```

Reply:
Changed to deletes the existing subscription for ...